### ISSUE TITLE
netplay got lags after update to 1.7.6

### ISSUE THREAD

**Issue by Yasaka7** [libretro/RetroArch#8273]
[Description of the bug]

i always play snes9x netplay with a friend. everything worked fine with 1.7.5 (and older versions) and latest snes9x kernel. after the update, there were strange lags. when he played, i had lags (never happened before) watching his gameplay and he said to me the music bugs for him.(i am the host) we downgraded to 1.7.5 and everything worked fine again.

### Steps to reproduce the bug

use latest 1.7.6 with latest snes9x core and play some mario world in netplay.

- RetroArch: 1.7.6
- OS: Windows 10

**Comment by orbea** [comment]
Would you be willing to bisect the issue to find the first bad commit?

**Comment by furiadeoso** [comment]
Hi, @Yasaka7. 

Besides bisecting, could you explain what you mean when say "lag" and "music bugs"? 
Could you also activate the fps and check if you or your friend do not have framerate drops?

Regards.

**Comment by Yasaka7** [comment]
@furiadeoso https://www.twitch.tv/videos/377105320?t=00h16m01s you can clearly see when he plays (lags) and when i play without lags. i dont know what my friend meaned by music bugs. probably some crackling or random noises. we dont  really have fps drops.

**Comment by furiadeoso** [comment]
@Yasaka7 It seems your fiend are not using the same Retroarch and/or core version as you.

**Comment by Yasaka7** [comment]
we checked it. we have the same version and the same snes9x core. he has input delay and lags when i play, and i have lags while watching when he plays.

**Comment by furiadeoso** [comment]
And other games and cores? Is present this "lag" only in 1.7.6?

**Comment by Yasaka7** [comment]
yes like i said this lag is only in 1.7.6. when we try the same snes9x core in 1.7.5 everything is just fine. we didnt test any other cores or games. we usually play smw/smw hacks via netplay.

**Comment by furiadeoso** [comment]
I'm going to try to reproduce it asap. Are you using the same Retroarch.cfg settings with 1.7.5 and 1.7.6?
Could you try to bisect the issue trying with nightly versions between stables?

**Comment by Yasaka7** [comment]
i think we are using the same cfg file. we just "updated" retroarch which means i extracted the content of 1.7.6 in the 1.7.5 directory and overwrote everything. uhm how many nightly versions are there? i dont think my friend wants to spend several hours testing nightlys... either i.

**Comment by fr500** [comment]
you're the only one who has seen the issue.
if you can't bisect then there is nothing to be done.

**Comment by furiadeoso** [comment]
Well. A good start would be try with another game or/and core, changing only the exe 1.7.5/6 .
And somebody more could try to reproduce the issue. I'll try when I can find a friend.

Nevertheless, if the issue is confirmed is needed to bisect the problem, because there are no significant netcode changes between these two releases.

**Comment by Yasaka7** [comment]
@furiadeoso we can try it if you want.

**Comment by furiadeoso** [comment]
@Yasaka7 
Tested. I have no problem: https://www.twitch.tv/videos/389644681 . If you consider there are any issue you must  bisect  and or do more tests.

Regards.

**Comment by furiadeoso** [comment]
BTW I was a recent nightly.

**Comment by Yasaka7** [comment]
so we tested it with a nightly build and a fresh config, no leftovers from an old version. it still lagged like described above. since we cant really bisect it, i think we just stand on 1.7.5. we updated the snes9x core there on 1.59.2 and it works really well. if we are the only ones with that issue i assume its some problem on our side and since it works on 17.5. perfectly there is not really a problem with not updating.
